cookbooks/fb_apache/recipes/default.rb

```diff
@@ -81,6 +112,16 @@
   notifies :restart, 'service[apache]'
 end
 
+# Remove unmanaged sites
+::Dir.glob("#{sitesdir}/*").reject {|e| ::File.directory?(e)}.each do |entry|
+  next if ['fb_sites.conf', 'fb_apache.conf'].include?(::File.basename(entry))
```

Comment:
You don't, but you do need:

```
Listen 443 https
SSLSessionCache         shmcb:/run/httpd/sslcache(512000)
SSLSessionCacheTimeout  300
SSLRandomSeed startup file:/dev/urandom  256
SSLRandomSeed connect builtin
SSLCryptoDevice builtin
```

Or for php:

```
<FilesMatch \.php$>
    SetHandler application/x-httpd-php
</FilesMatch>
AddType text/html .php
DirectoryIndex index.php
php_value session.save_handler "files"
php_value session.save_path    "/var/lib/php/session"
```

Without which PHP basically doesn't work... and the distro keeps this stuff up-to-date. @davide125  and I had talked about nuking this stuff and making people put it in their `node['fb_apache']['extra_configs']` but it seemed a bit excessive...